.github/workflows/points-next.yml

```diff
@@ -0,0 +1,55 @@
+name: Points with next Rails
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  build-rails-next:
+    runs-on: ubuntu-latest
+    
+    env:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      RAILS_ENV: test
+    
+    services:
+      postgres:
+        image: postgres:11.6-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+
+      - name: Install PostgreSQL 11 client
+        run: |
+          sudo apt-get -yqq install libpq-dev
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.0.2"
+
+      - name: Build App
+        run: |
+          bundle install
+          yarn install
+          cp config/database.yml.github config/database.yml
+          bundle exec rails db:create
+          bundle exec rails db:schema:load
+          gem install next_rails
+          next bundle install 
```

Comment:
since this project is running agains Rails' main branch, we should update the rails version or it will get stuck always testing against the git commit that's in the current .next.lock, we do that in dash https://github.com/fastruby/dash/blob/main/.circleci/config.yml#L114